universal-application-tool-0.0.1/app/services/applicant/SummaryData.java

```diff
@@ -0,0 +1,31 @@
+package services.applicant;
+
+public class SummaryData {
+  /** The question text */
+  public String questionText;
+
+  /** The applicant's response to the question. */
+  public String answerText;
+
+  /** The block id where this question resides. */
+  public String blockId;
```

Comment:
(blocking maybe?): A `blockId` without a `programId` is a bit scary since a block id doesn't mean much without a program id. I get this is used for generating links, but it doesn't seem to match the abstraction (at least the one in my head). 

Would it make sense to have an `Answer` class, and then another class that has an `Answer` that used for views.

Reply:
Well, I suppose the link needs the program id as well, but I left that out because it'd just be repeated across all objects in the list. For data completeness within a single entity I suppose it'd probably be best to add the program id here.